lib/query/query_supporters.rb

```diff
@@ -231,6 +231,8 @@ def self.full_filter_expr(np_id, query)
     if query[:search].present?
       expr = expr.and_where(%Q(
         supporters.fts @@ websearch_to_tsquery('english', $search)
+        OR supporters.name ILIKE $search
```

Comment:
I think that's the reason our name search got worse, we used to compare using `supporters.name ILIKE $search`, but after we added the full text search index, we removed that.

Reply:
We won't need that if the next line is correct, though, I added this back to test.